app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    com = command.split()[0]
+    file_name = command.split()[1]
+    copy_name = command.split()[2]
```

Comment:
you can unpack these variables in the easiest way, also use the more descriptive names
`command, old_file, new_file = command.split()`